openfisca_core/variables.py

```diff
@@ -4,11 +4,11 @@
 import inspect
 import textwrap
 
-from openfisca_core.formulas import SimpleFormula, DatedFormula, new_filled_column
+from openfisca_core.formulas import Formula, new_filled_column
 
 
-class AbstractVariable(object):
-    formula_class = None
+class Variable(object):
+    formula_class = Formula
```

Comment:
formula_class removed from Variable but maintained as item of Formula.dated_formulas_class